apps/frontend/src/libs/components/modal/modal.module.css

```diff
@@ -0,0 +1,74 @@
+.modal-overlay {
+	position: fixed;
+	z-index: 1000;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	width: 100%;
+	height: 100%;
+	background: var(--color-overlay);
+}
+
+.modal-content {
+	position: relative;
+	z-index: 1;
+	width: 400px;
+	max-width: 90%;
+	padding: 20px;
+	color: var(--color-text-primary);
+	background-color: var(--color-background-secondary);
+	border-radius: 8px;
+	box-shadow: var(--box-shadow);
+}
+
+.modal-header {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	margin-bottom: 20px;
+}
+
+.modal-header h2 {
+	margin: 0;
+	font-size: 20px;
+	font-weight: bold;
```

Comment:
We use font-weight bold neither in design nor in the code, use number value instead of a word and it should be translated properly, in figma it's semi-bold